pkg/service/repair/export_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2023 ScyllaDB
```

Comment:
If I understand correctly, this file is created to have a possibility of invoking 
`(pm *dbProgressManager) setPrevRunID(ctx context.Context, ageMax time.Duration) `
outside of the `repair` package.

Integration tests are black-box ones, and they are part of `repair_test` package, so to let them to invoke what you needed, you put a wrapper over needed function and place it inside `_test` postfixed file. It means that export_test will not be part of the output binary.

It's clever, but hard to understand at the first glance.

I suggest to either:
- add the comment explaining the need for this file 
- just export `dbProgressManager` and the method you need `setPrevRunID`

Reply:
I actually like the export approach more.
I added a commit that exports ProgressManager interface alongside with NewDBProgressManager function.
This also makes the code look cleaner and make the interface more straight forward.